vnets.tf

```diff
@@ -82,6 +80,13 @@ resource "azurerm_virtual_network" "trusted" {
   address_space       = ["10.252.0.0/21"] # 10.252.0.1 - 10.252.7.254
   tags                = local.default_tags
 }
+resource "azurerm_virtual_network" "public_db" {
```

Comment:
this seems overly complicated.

The [document](https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-networking) is confusingly written.

you can have a vNET with private access without having to create another network and use peering.

Just set your connectivity method to private access 

<img width="716" alt="image" src="https://github.com/jenkins-infra/azure-net/assets/21194782/f1b7c70d-85e9-4fc2-af6f-90833860c3f6">

Reply:
I see would have been clearer if the description linked to that comment instead =/

Reply:
True that, good point. I wrote the PR too quickly yesterday. Let me update the body before merging